openapi.yaml

```diff
@@ -471,6 +591,173 @@ paths:
           $ref: "#/components/responses/InternalServerError"
 components:
   schemas:
+    AddRelationArgs:
```

Comment:
```suggestion
    IntegrateArgs:
```

Reply:
This was marked as resolved, but `IntegrateArgs` still seems to have the old schema.

Reply:
Sorry I missed this, fixing.. I think the provider/requirer duo makes sense :+1: